### PR TITLE
refactor: migrate from `slice.items` to `slice.primary` groups

### PR DESCRIPTION
**Pull request by lihbr**
Contributes to: [DT-2079](https://linear.app/prismic/issue/DT-2079/aadevuser-i-can-see-the-next-starters-updated-with-the-slice-model)

### Context

This PR migrates slices from using `slice.items` array to using regular group fields within the `slice.primary` section.

### Before Merging

- [x] Update content dump (WIP Michel)
- [x] Update repositories on production (once backend is on production)
- [x] Revert `slicemachine.config.json`

### CODE REVIEW

**Review by levimykel — APPROVED** [review]
Everything looks good to me. I left one small comment, but I approve the PR.

File: src/slices/Hero/model.json [hunk 1]
```diff
@@ -45,37 +45,45 @@
...
+                  "label": "Button Link",
+                  "placeholder": "Link for the button",
+                  "select": null
+                }
+              },
+              "text": {
+                "type": "Text",
+                "config": {
+                  "label": "Button Text",
+                  "placeholder": "Optional - Text for the button"
```
[the remark this answers is not in the document]
Comment (lihbr) [reply src/slices/Hero/model.json:67]: Updated on both crash course 🙏

[the remark this answers is not in the document]
Comment (levimykel) [reply src/slices/Hero/model.json:67]: You updated the wrong text field placeholder. I think the other change is good, but you can still do this one.

**Review by levimykel — APPROVED** [review]

File: src/slices/Hero/model.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (lihbr) [reply src/slices/Hero/model.json]: Done, thanks for checking again, don't know what I was doing 🤦‍♀️

**Pull request merged**